src/test/resources/data/InitialiseReturnsSubmissionPayload.Json

```diff
@@ -0,0 +1,5 @@
+{
+  "totalRecords": 1000,
```

Comment:
Considering this has a parameter token that you replace - this could just be a case class with a custom apply method that only takes in the current tax year as a parameter

`def apply(currentTaxYear: Int) { InitialiseReturnsSubmissionPayload(100, "APR", currentTaxYear)`